ruby/install/run.sh

```diff
@@ -7,15 +7,26 @@ if [ ! ${IS_COLOR_SOURCED} ]; then
 fi
 
 echo "${GREEN}Installing... ${BROWN}ruby...${NC}"
-gpg --keyserver hkp://keys.gnupg.net --recv-keys 409B6B1796C275462A1703113804BB82D39DC0E3 7D2BAF1CF37B13E2069D6956105BD0E739499BDB
-curl -sSL https://get.rvm.io | bash -s stable --ruby
-gem install grb --no-document
+
+echo "${RED}Which ruby version manager would you like to use? (See readme for details) ${GREEN}[rbenv,rvm] default: rbenv${NC}"
+read choice
+
+case $choice in
+    rvm) source ${DIR}/install_rvm.sh;;
+    rbenv) source ${DIR}/install_rbenv.sh ;;
+    *) source ${DIR}/install_rbenv.sh ;;
```

Comment:
can you make it default to rvm, that's what we use internally currently and most dev have it installed :)

Reply:
done.